tensorboard/webapp/core/views/page_title_container.ts

```diff
@@ -96,6 +99,8 @@ export class PageTitleContainer {
 
   constructor(
     private readonly store: Store<State>,
-    @Optional() @Inject(TB_BRAND_NAME) private readonly customBrandName: string
+    @Optional()
+    @Inject(TB_BRAND_NAME)
+    private readonly customBrandName: string | null
```

Comment:
Just wondering, for an `Optional` param, it is `undefined` and not `null`, right?

Reply:
Fixed. Thanks for the lightning-fast response!

Reply:
Just jumping in to double check, does `@Optional` actually return `undefined` when the dependency is not provided?  These official Angular docs suggest it returns `null`, so not sure if the Angular site is just outdated:

"The @Optional property decorator tells Angular to return null when it can't find the dependency."
https://angular.io/guide/dependency-injection-in-action#make-a-dependency-optional-and-limit-search-with-host
https://angular.io/api/core/Optional

Reply:
This is fixed in #5126, thanks Stephan!